trytond/backend/postgresql/database.py

```diff
@@ -331,12 +334,18 @@ def get_connection(self, autocommit=False, readonly=False):
             conn.set_isolation_level(ISOLATION_LEVEL_AUTOCOMMIT)
         else:
             conn.set_isolation_level(ISOLATION_LEVEL_REPEATABLE_READ)
+        statements = []
```

Comment:
Au passage, d'après la doc, `set_isolation_level` c'est deprecated

Reply:
C'est l'objet d'un changement que Cédric a fait ce WE qui vire le support de psycopg2cffi (qui était cassé).